spec/requests/api/admin/search_references_controller_spec.rb

```diff
@@ -0,0 +1,41 @@
+RSpec.describe Api::Admin::SearchReferencesController, type: :request do
+  before do
+    create :search_reference, referenced: create(:heading), title: 'aa'
+    create :search_reference, referenced: create(:chapter), title: 'bb'
+    create :search_reference, referenced: create(:commodity), title: 'bb'
+  end
+
+  describe 'GET #index' do
+    subject(:do_request) do
+      authenticated_get api_search_references_path(params: query_letter, format: :json)
```

Comment:
These path helpers are awful. Missing the admin scope you might confuse it with the v2/v1 apis :facepalm: